esmvaltool/utils/cmorizers/obs/cmor_config/Landschutzer2014.yml

```diff
@@ -0,0 +1,42 @@
+---
+# project at hand
+proj:
+  dataset: 'Landschutzer2014'
+  version: 'CDIAC_G05'
+  realm: 'clim'
+  field: 'TO2M'
```

Comment:
Field is no longer used
```suggestion
```

Reply:
This is a leftover from the previous PR and I will clean it.